### PR TITLE
LPS-153782 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-153782

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase [hunk 1]
```diff
@@ -363,6 +363,51 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-147579. View expired message of web content on Content Display fragment."
+	@priority = "4"
+	test ViewExpiredWebContentMessage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase:368]: ViewExpiredWebContent

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase [hunk 2]
```diff
@@ -363,6 +363,51 @@ definition {
...
+			}
+
+			task ("Select the web content in Content Display") {
+				PageEditor.editContentDisplay(
+					fragmentName = "Content Display",
+					webcontent = "true",
+					webContentContent = "Web Content Content",
+					webContentTitle = "Web Content Title");
+			}
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase:394]: Add step to pubish the content page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase [hunk 3]
```diff
@@ -363,6 +363,51 @@ definition {
...
+
+		task ("When the page creator expires the web content") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContent.expireCP(webContentTitleList = "Web Content Title");
+		}
+
+		task ("Then the page creator should see an expired message on Content Display") {
+			ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase:403]: Navigate to the view mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase [hunk 4]
```diff
@@ -363,6 +363,51 @@ definition {
...
+				pageName = "Content Page Name",
+				siteName = "Test Site Name");
+
+			Alert.viewWarningSpecific(warningMessage = "Web Content Title is expired.");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplay.testcase:407]: Try to use 	WebContentDisplayPortlet.viewMessage(expiredWebContentTitle = "Web Content Title");

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comment? Thanks

**Comment by ozysouza** [comment]
ci:forward

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comment? Thanks

Hi @Tim-Cao! I've pushed a new commit with the change and forward it. Thanks!

**Pull request closed without merging**